### ISSUE TITLE
Edges only listed unidirectional in vertices

### ISSUE THREAD

**Issue by fixundfertig123** [orientechnologies/orientdb#6401]
## Expected behavior and actual behavior

Expected behavior:
An edge connecting two vertices should be visible in both vertices as either in or out property.

Actual behavior:
As depicted in the picture below I have two vertices #20:0 and #20:3 connected to each other via the edges #24:452 (#20:3 -> #20:0) and #24:453 (#20:0 -> #20:3). However only vertex #20:3 lists the edges as in / out edge? Recreating fails, since a Unique out/in index prevents orientdb from creating duplicates. 

From a first inspection it looks like the OMultiCollectionIterator used in OrientVertex.getEdges does not return the proper amount of edges.
## Steps to reproduce the problem

To be done, since the error just occurred (at lease to my current understanding) after the migration to 2.2.3.
## Important Questions

The edge should be symmetrically visible from both vertices. 
### Runninng Mode
- [ ] Embedded, using PLOCAL access mode
- [ ] Embedded, using MEMORY access mode
- [x] Remote (localhost)
### Misc
- [ ] I have a distributed setup with multiple servers. How many? 
- [ ] I'm using the Enterprise Edition
### OrientDB Version
- [ ] v2.0.x - Please specify last number: 
- [ ] v2.1.x - Please specify last number: 
- [x] v2.2.x - Please specify last number: 3
### Operating System
- [ ] Linux
- [ ] MacOSX
- [x] Windows
- [ ] Other Unix
- [ ] Other, name? 
### Java Version
- [ ] 6
- [ ] 7
- [x] 8

![image](https://cloud.githubusercontent.com/assets/5039358/16697598/b6fa4c30-454a-11e6-8017-ffb26a9543fa.png)

**Comment by tglman** [comment]
hi @fixundfertig123,

from the picture i actually see the links of #20:0 but not the one of #20:3, am i wrong? 

Just to be sure that i got your problem.

Regards.

**Comment by fixundfertig123** [comment]
Hi @tglman,

your right. Seems like the edges exist but are internally not linked to the vertices.

I am currently working on isolating the error, unfortunately no success yet. Good think is I can reproduce it. Bad think I cannot just publish it, since  would need to publish quite a bit of my code. Is there anything I can do for you?

Thanks & Cheers

**Comment by tglman** [comment]
hi @fixundfertig123,

If you can produce an test example independent from your code, or maybe just a list of steps for reproduce the problem, it will be very helpful.

**Comment by tglman** [comment]
hi @fixundfertig123, 

did you have chance to try a later hotfix?  this problem may have been already fixed.

Regards

**Comment by fixundfertig123** [comment]
Hi @tglman,
sorry for not replying earlier. I did not have a chance to test the later hotfixes. 

The problem had something to do with detached objects (So my fault, sorry for that!!) in my code. I could get my orignal problem fixed by doing "reloads" after the attachments. However, I am still confused why I got inconsistent state from the graph editor (Vertex #20:3 not showing the edges in/outs).

So, shall I close the issue?

Cheers, Hanno
